content/en/project/tulpa/index.md

```diff
@@ -0,0 +1,329 @@
+</div>
+
+## My to-do list 
+* [X] preprocess the fMRI data with fmriprep
+* [X] run GLMs using SPM for the first level and FSL Randomise for the second level 
+* [X] replicate GLM in Nilearn 
+   * [X] replicate a GLM in Nilearn 
+   * [X] compute the correlation between first-level beta maps in the SPM and Nilearn analysis 
+   * [ ] replicate the Nilearn GLM with the exact same parameters as in SPM 
+      * fd_threshold = 1 (default is 0.5mm)
+      * change from `load_confound_strategy` to `load_confound`as it's more custamizable and allows to match parameters of SPM.
+      * check if I used a high pass filter in SPM, if not deactivate it in Nilearn.
+      * check if I standardized in SPM, if not deactivte it in Nilearn.
+   * [ ] Cluster correct the Nilearn second-level contrasts using [`cluster_level-inference`](https://nilearn.github.io/dev/modules/generated/nilearn.glm.cluster_level_inference.html#nilearn.glm.cluster_level_inference).
+   * [ ] re-run the GLMs including a brain mask that excludes the cerebellum and brain stem. As we are not interested in the analysis of these regions their exclusion is legit. I can use a mask from brainvolt that I can [download here](https://neurovault.org/collections/6196/).
+* [X] connectivity analysis 
+   * [X] compute connectome for all conditions and save as `.npy`
+   * [ ] compute `graph theory` metrics to characterise and compare the connectomes. François Paugam recommended to look into the [Networks of the Brain](http://www.cs.cmu.edu/~saketn/files/Sporns_Book.pdf) as an introduction.
+* [X] compute seed-to-voxel correlations 
+* [X] ML classifier 
+   * [X] ML classifier on connectomes 
+   * [ ] ML classifier on seed-to-voxel maps 
+   * [ ] ML classifier on the average activity for each of the 10 trials (not entire runs) cropped to an ROI in the SMA
+* [X] Deep Neural Network Decoder using PyTorch
+
+## Feedback from the final presentation 
+- visualise things in MRIcroGL, soon implemented in nilearn as `niview`
+- change my pearson-r values to [-1, 1] (currently [-100,100])
+- for comparing nilearn and spm beta maps: specify not-matched. E.g.:
+   - between software, within subject, within contrast, within run 
+   - between software, between subject, between contrast, between run 
+- feature selection must be either on (A) train set or (B) integrated in cross-validation. Make sure it's not selecting featuers on the entier dataset --> danger of overfitting!
+- r2 is a measure for the percentage of variance explained. Not relevant here, thus, remove it. 
+- explain permutation test for significance testing.
+- apply the classifer on beta maps per trials (there are 10 trials per run). I can use ROIs or entire beta masks. The model will take care of the many variables. 
+- try a searchlight appraoch: train a classifer on different ROIs or voxels and then compare the accuracy. This could lead to a accuracy map with voxel resolution, however, this is computationally expensive. 
```

Comment:
```suggestion
- try a searchlight approach: train a classifer on different ROIs or voxels and then compare the accuracy. This could lead to an accuracy map with voxel resolution, however, this is computationally expensive. 
```
typos